llvm-runtimes/libatomic-stub/libatomic-stub-1.0.0.ebuild

```
@@ -0,0 +1,23 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Stub library which allows compiler-rt to be used as an libatomic"
```

```suggestion
DESCRIPTION="Stub library which allows compiler-rt to replace libatomic"
```

"An libatomic" doesn't sound grammatically correct, also it's kinda long.